tests/receivers/kong/testdata/all_metrics_config.yaml

```diff
@@ -0,0 +1,24 @@
+receivers:
+  prometheus:
```

Comment:
since this uses the prometheus receiver, I think it belongs in that directory.

Reply:
moved